src/organisms/cards/PolicyCard/PolicyInformation.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import accounting from 'accounting';
+import Hide from 'wrappers/Hide';
+import Text from 'atoms/Text';
+import styles from './policy_card.module.scss';
+
+export const PolicyInformation = ({ financialStrength, customerService, totalCustomers }) =>
+  <Hide hideOn='small medium' className={styles['policy-info']}>
+    <Text
+      type={7}
+      color='neutral-3'
+      semibold
+    >
+      <div className={styles['label']}>Financial Strength</div>
+      <Text color='neutral-2' semibold>{financialStrength}</Text>
+    </Text>
+    <Text
+      type={7}
+      color='neutral-3'
+      semibold
+    >
+      <div className={styles['label']}>Customer Service</div>
+      <Text color='neutral-2' semibold>{customerService}</Text>
+    </Text>
+    <Text
+      type={7}
+      color='neutral-3'
+      semibold
+    >
+      <div className={styles['label']}>Total Customers</div>
+      <Text color='neutral-2' semibold>{accounting.formatNumber(totalCustomers)}</Text>
```

Comment:
@cisacke maybe better to offload this formatting for implementation? what do you think?

Reply:
yeah good point - I think if there was uncertainty around the content of the information panel we'd want to keep the props as agnostic as possible but since the labels are hardcoded, if product decides to change anything we'd have to go back into the component and update. I'm gonna keep it as is for that reason but possible in the future we could condense the three props to an array of objects

Reply:
sounds good